packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -429,12 +429,10 @@ class LightCurveViewer extends Component {
     const { onKeyDown } = props
 
     const container = this.svgContainer.current
-    const containerHeight = `${container.clientHeight}px`
-
     this.d3svg = d3.select(container)
       .append('svg')
       .attr('class', 'light-curve-viewer')
-      .attr('height', containerHeight)
```

Comment:
I did a bit more inspection, and I think we could just set height back to a `vh` unit like https://github.com/zooniverse/front-end-monorepo/pull/3139/files. For instance `60vh` or `70vh`. When I run this branch locally and change height to 70vh, the axis is not hidden.

Reply:
Wait I lied - I just tried using `vh` units again and the axis does not behave as we want it to 😞

Reply:
Unfortunately that would regress the problems that @shaunanoordin mentions here: https://github.com/zooniverse/front-end-monorepo/pull/3139#pullrequestreview-972365646